2021-04-20 14:22:53,631 INFO    MainThread:6228 [wandb_setup.py:_flush():69] setting env: {}
2021-04-20 14:22:53,631 INFO    MainThread:6228 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 14:22:53,631 INFO    MainThread:6228 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210420_142253-2obilq9n/logs/debug.log
2021-04-20 14:22:53,631 INFO    MainThread:6228 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210420_142253-2obilq9n/logs/debug-internal.log
2021-04-20 14:22:53,632 INFO    MainThread:6228 [wandb_init.py:init():369] calling init triggers
2021-04-20 14:22:53,632 INFO    MainThread:6228 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-20 14:22:53,632 INFO    MainThread:6228 [wandb_init.py:init():418] starting backend
2021-04-20 14:22:53,632 INFO    MainThread:6228 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-20 14:22:53,644 INFO    MainThread:6228 [backend.py:ensure_launched():123] starting backend process...
2021-04-20 14:22:53,648 INFO    MainThread:6228 [backend.py:ensure_launched():128] started backend process with pid: 6246
2021-04-20 14:22:53,650 INFO    MainThread:6228 [wandb_init.py:init():423] backend started and connected
2021-04-20 14:22:53,652 INFO    MainThread:6228 [wandb_init.py:init():465] updated telemetry
2021-04-20 14:22:53,652 INFO    MainThread:6228 [wandb_init.py:init():484] communicating current version
2021-04-20 14:22:54,421 INFO    MainThread:6228 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-20 14:22:54,422 INFO    MainThread:6228 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-20 14:22:54,630 INFO    MainThread:6228 [wandb_init.py:init():522] starting run threads in backend
2021-04-20 14:22:54,653 INFO    MainThread:6228 [wandb_run.py:_console_start():1524] atexit reg
2021-04-20 14:22:54,653 INFO    MainThread:6228 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-20 14:22:54,654 INFO    MainThread:6228 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-20 14:22:54,656 INFO    MainThread:6228 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-20 14:22:54,656 INFO    MainThread:6228 [wandb_init.py:init():546] run started, returning control to user process
2021-04-20 14:23:04,984 INFO    MainThread:6228 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-20 14:23:04,985 INFO    MainThread:6228 [wandb_run.py:_restore():1466] restore
2021-04-20 14:23:07,221 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 539
  total_bytes: 539
}

2021-04-20 14:23:07,642 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 539
  total_bytes: 3388
}

2021-04-20 14:23:08,055 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:08,157 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:08,259 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:08,361 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:08,464 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:08,566 INFO    MainThread:6228 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3388
  total_bytes: 3388
}

2021-04-20 14:23:09,665 INFO    MainThread:6228 [wandb_run.py:_show_files():1838] logging synced files
